lwt/nbd_lwt_unix.mli

```diff
@@ -22,7 +22,7 @@ type tls_role =
 
 val connect: string -> int -> Channel.channel Lwt.t
 (** [connect hostname port] connects to host:port and returns
-    a channel. *)
+    a generic_channel which does not and will not use TLS. *)
```

Comment:
`[generic_channel]` could be written like this, to render it as code, because it's a type. See http://caml.inria.fr/pub/docs/manual-ocaml/ocamldoc.html#sec339 for ocamldoc comment formatting.